serverless.yml

```diff
@@ -173,6 +173,7 @@ stepFunctions:
                     id.$: $.id
                     type.$: $.type
                     dataType.$: $.out.dataType
+                  ResultPath: $.transformResult
```

Comment:
Were also going to change the `ResultPath` coming out of the `loadToObservationDb`? It's not a going to hold up the this PR either way.

Reply:
Yeah let's do it.  I had initially left it out to limit the number of changes, but I think it helps to clarify.